src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java

```diff
@@ -34,6 +37,17 @@ public DisplayCommInterfaceImplementation(DisplayComm2WayInterface displayComm2W
   * @return the TrackData
   */
   public TrackData receiveRfidRequest(int rfidNum) {
-    return comm2Way.passRfidRequest(rfidNum);
+    this.data = comm2Way.passRfidRequest(rfidNum);
+
+    System.out.println("Track Data for RFID tag number " + rfidNum);
+    System.out.println("Location Data Points: " + data.getLocationDataPoints());
+    System.out.println("Course Direction: " + data.getCourseDirection());
+    System.out.println("Speed: " + data.getSpeed());
+    System.out.println("Status: " + data.getStatusMessage());
+
+    this.plotter = new TrackDataPlotter(data);
```

Comment:
If we remove the above private class attributes, then this gets the type added to it.
```suggestion
    TrackDataPlotter plotter = new TrackDataPlotter(data);
```